mpz-core/src/lpn.rs

```diff
@@ -0,0 +1,180 @@
+//! Implement LPN with local linear code.
+//! More especifically, a local linear code is a random boolean matrix with at most D non-zero values in each row.
+
+use crate::{prp::Prp, Block};
+use rayon::prelude::*;
+/// An LPN encoder.
+///
+/// The `seed` defines a sparse binary matrix `A` with at most `D` non-zero values in each row.
```

Comment:
@xiangxiecrypto , is **at most** correct here? Shouldn't it be **exactly** D non-zero values?

Reply:
It is **at most**. Recall that the process is choosing `d` positions at random in a row, and filling the `d` positions with random elements (in F_2). You do not need to fill all `d` positions with `1`.